app/build.gradle

```diff
@@ -105,8 +105,8 @@ android {
         viewBinding true
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
```

Comment:
Is this change from 1.8 to 17?!

Reply:
Lol, yup. BUT: Java's naming system changed in between Java 1.8 is essentially Java 8 in the current naming system, with 8, 11 and 17 LTS releases. Not sure what happened to 14 and why that never became an LTS.